RadixWallet/Features/GatewaySettingsFeature/GatewaySettings+Reducer.swift

```diff
@@ -38,13 +38,15 @@ public struct GatewaySettings: Sendable, FeatureReducer {
 	}
 
```

Comment:
Might as well go all the way and add the `@Reducer` macro, so we stay up to date with what TCA adds in the future. Note the things you need to make explicit for that to work. [See here](https://rdxworks.slack.com/archives/C03QFAWBRNX/p1722331252960899).